app/templates/teachers/teacher-student-view.jade

```diff
@@ -364,15 +364,18 @@ mixin breadcrumbs
                               .small!= marked(solution.description || '')
                           tr
                             td
-                              if view.levelStudentCodeMap[level.get('original')]
+                              - var multipleCodes = view.levelStudentCodeMap[level.get('original')].length > 1
```

Comment:
`view.levelStudentCodeMap[level.get('original')]` is used three time in short succession. It would be cleaner to use a local variable here to shorten it. Suggest `levelStudentCodes` or `matchingCodes` or even just `codes`.

While on the topic of variable names, I think `codemap` is misleading, it's just `code`, right?

Reply:
right!